_blogs/2015-02-15-example-1.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Example 1
```

Comment:
Is this required for production?

Reply:
I will keep one example.md ( will get rid of the other twos)  file to facilitate using markdown format for blogs( as all of the previous ones are HTML files). However setting published: false allows us to decide whether or not this will appear on website.